### PR TITLE
Login option added

### LINKED ISSUE #5: Login and logout should show corresponding icons
Login and logout should show different  icons
At present only logout icon is shown

### PR DESCRIPTION
**Pull request by Nish-19**
Fixes #5 
New login option added in menu drawer

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ apply from: './jacoco.gradle'
 android {
 
     compileSdkVersion rootProject.compileSdkVersion
-    buildToolsVersion rootProject.buildToolsVersion
+    buildToolsVersion '28.0.3'
```
Comment (dr0pdb) [inline app/build.gradle]: please revert this change.

File: build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.4'
+        classpath 'com.android.tools.build:gradle:3.2.1'
```
Comment (dr0pdb) [inline build.gradle]: please revert this change.

**Review by dr0pdb — CHANGES_REQUESTED** [review]
Sorry for the late review. Please revert the unnecessary changes.

Also provide a screenshot of how it looks in the app.

**Comment by dr0pdb** [comment]
@Nish-19 Please provide a screenshot of how it looks.

**Comment by Nish-19** [comment]
image

- In spite of updating the login and logout option in the xml file,the app shows login text for both the login and logout options as shown in the screenshot.


- However in the android studio it displays it correctly as you can find in the screenshot below.
image

**Comment by dr0pdb** [comment]
@Nish-19 If it is showing Login for both, then there must be an issue with the program right? we have to fix it. I can't merge the PR before that.

**Comment by Nish-19** [comment]
@srv-twry Can you give me an idea as to how do I go about this problem? Changing the xml file doesn't seem to be enough. I think I'll have to make a few other changes in other files. I hope you can provide me information on how to go about this.

**Pull request closed without merging**